src/workers/parser/tests/mapOriginalExpression.spec.js

```diff
@@ -3,31 +3,35 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 import mapOriginalExpression from "../mapOriginalExpression";
+import { parseConsoleScript } from "../utils/ast";
 import { format } from "prettier";
 
 const formatOutput = output =>
   format(output, {
     parser: "babylon"
   });
 
+const mapOgExpression = (expression, mappings) =>
+  mapOriginalExpression(expression, parseConsoleScript(expression), mappings);
```

Comment:
maybe it would be better to use the public API `mapExpression`?

Reply:
done